.gitignore

```diff
@@ -130,3 +130,4 @@ dmypy.json
 .pyre/
 
 # End of https://www.gitignore.io/api/python
+/.idea/
```

Comment:
Why?

Reply:
this is for PyCharm detritus from the project and index files

Reply:
Ok, thanks for the clarification!